packages/browser/src/tracekit.ts

```diff
@@ -32,9 +17,7 @@ export interface StackFrame {
 export interface StackTrace {
   name: string;
   message: string;
-  mechanism?: string;
   stack: StackFrame[];
```

Comment:
Eventually I just want this to return `Exception` since that contains everything we need. 

Currently working on an issue to cover these PRs and general approach/direction.